packages/cashier/src/pages/deposit/crypto-deposit/crypto-deposit.tsx

```diff
@@ -128,6 +143,9 @@ const CryptoDeposit = observer(() => {
 
     return (
         <div className='cashier__wrapper crypto-deposit__wrapper'>
+            <div className='crypto-deposit__header'>
+                <Breadcrumb items={crumbs} handleOnClick={onBreadcrumbHandler} />
+            </div>
```

Comment:
```suggestion
            <CryptoDepositBreadcrumb />
```

@heorhi-deriv Can we move the whole thing into a separate component?
`CryptoDeposit` is already big, I think we can isolate this `Breadcrumb` logic into a new component as it's not relaying on anything here 🤔